tools_docker/Debian_Uwsgi_ARM64/entrypoint.sh

```diff
@@ -197,6 +197,25 @@ if [ "$REPLY_BY_EMAIL" = "1" ];then
     log "Starting mail fetcher"
     supervisorctl start tracim_mail_fetcher
 fi
+
+## Check running services
+REDIS_SERVICE_RUNNING=$(ps -ef | grep -v grep|grep "redis-server" | wc -l)
+if [ $REDIS_SERVICE_RUNNING = 0 ]; then
+    logerror "Redis service not start. Mandatory!!"
+    exit 1
+fi
+PUSHPIN_SERVICE_RUNNING=$(ps -ef | grep -v grep|grep "pushpin" | wc -l)
+if [ $PUSHPIN_SERVICE_RUNNING = 0 ]; then
+    logerror "Pushpin service not start. Mandatory!!"
```

Comment:
```suggestion
    logerror "Pushpin service must be running.""
```

Reply:
Carefull, there is 2 " at the end if you apply this suggestion